src/printer.js

```diff
@@ -59,12 +59,16 @@ const {
   normalizeMagicMethodName
 } = require("./util");
 
+function isSupported(actualVersion, requiredVersion) {
```

Comment:
Being a bit nitpicky again: I think this function tells nothing about anything being supported, it just compares version strings. I'd rename it to something like `isMinVersion`.

That's debatable though.

Reply:
Agree & fixed.